src/styles/themes/Theme.tsx

```diff
@@ -0,0 +1,34 @@
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+import 'styles/fonts.scss';
+
+interface Props {
+  children: React.ReactNode
+}
+
+const theme = createTheme({
+    components: {
+      MuiAppBar: {
+        styleOverrides: {
+          root: {
+            backgroundColor: '#16bac6',
+          },
+        },
+      },
+      MuiButton: {
+        styleOverrides: {
+          root: {        
+            fontFamily: '"Ubuntu", sans-serif',
+            textTransform: 'capitalize',
+            color: '#112025',
+            fontSize: '1rem'
+          },
+        },
+      },
+    },
+  });
+
+  const Theme = ({ children }: Props) => (
```

Comment:
export const Theme =

Reply:
> > I like it! Maybe it would be nice to leave the hamburger and put it on the right side, remove the word PUPPYLOG and remove the bottom line in the navigation, as in the figma project?
> 
> it looks like this on a figma, logo on left and hamburger menu on right ![obraz](https://user-images.githubusercontent.com/56917808/150114738-baed6d03-26ae-4d91-9182-3dcf6e6f162e.png) Now, logo is in progress by ania. If will be the end I change to prepare from Ania
 But I change without color. Let it be as in figma

>